app/src/main/java/com/example/bottomnav/MainActivity.java

```diff
@@ -0,0 +1,34 @@
+package com.example.bottomnav;
+
+import android.os.Bundle;
+import com.google.android.material.bottomnavigation.BottomNavigationView;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.navigation.NavController;
+import androidx.navigation.Navigation;
+import androidx.navigation.ui.AppBarConfiguration;
+import androidx.navigation.ui.NavigationUI;
+import com.example.bottomnav.databinding.ActivityMainBinding;
+
+public class MainActivity extends AppCompatActivity {
+
+private ActivityMainBinding binding;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+     binding = ActivityMainBinding.inflate(getLayoutInflater());
+     setContentView(binding.getRoot());
+
+        BottomNavigationView navView = findViewById(R.id.nav_view);
```

Comment:
```suggestion
```

This can be deleted: this isn't used (since `navView` is provided by `binding.navView`).